### ISSUE TITLE
Can I somehow show a few rewarded ads in a row?

### ISSUE THREAD

**Issue by alectogeek** [AndriousSolutions/ads#34]
Is it possible to pre-download a few rewarded ads? As I know, the native SDKs provide such a feature, but I cannot find any information related to this feature in ads documentation

**Comment by Andrious** [comment]
' pre-download a few rewarded ads?'  I'm not aware if the plugin, firebase_abmob, offers such an ability. Would you describe how such a feature would be used?

**Comment by alectogeek** [comment]
Yes, you're right, the firebase_admob doesn't provide such ability, but native SDKs do, as far as I know. 
I have a button in my app giving the user additional life (like in a game). So, when the user presses the button, watches the ad and gets back to my app, he sees the button again. For now, if he presses the button right after watching the previous ad, he has to wait some time because a new ad us being downloaded.

**Comment by Andrious** [comment]
I think I understand. Without viewing the particular code, could not an asynchronous operation load a new ad while watching the first ad---allowing the next press of the button to be more responsive?

**Comment by alectogeek** [comment]
Yeah, right. 
In other words: it would be cool to be able to have the next already cached rewarded ad once user have closed the previous one.

**Comment by Andrious** [comment]
Would you give me an idea of how you're using the Ads package in your app?
Below is oversimplified, and so give me an idea of how you've set things up.

```
    // Initialize
    _ads = Ads(
      appId,
      videoUnitId: videoUnitId,
    );




   // To show a video
  _ads.showVideoAd();

```

**Comment by alectogeek** [comment]
Oh, sorry for my late response!
Yes, I use exactly as you wrote, nothing extraordinary. The howVideoAd() is being triggered when a user in my app presses "Get additional life". The main problem is that as soon as the user has watched one video and got his additional life, he can press the button immediately to get one more
